libs/auth-api-lib/seeders/data/scope-passkeys.ts

```diff
@@ -0,0 +1,17 @@
+import { compose, createScope } from './helpers'
+
+export const up = compose(
+  createScope({
+    name: '@island.is/auth/passkeys',
+    displayName: 'Aðgangslyklar',
+    description:
+      'Gerir notanda kleift að innskrá sig með lífkenni í gegnum aðgangslykil',
+    addToResource: '@island.is',
+    addToClients: ['@island.is/app', '@island.is/web'],
+    delegation: {
```

Comment:
We probably need to add entries to the `api_scope_delegation_types` table when creating scopes this way.